alias.go

```diff
@@ -0,0 +1,50 @@
+package analytics
+
+import "time"
+
+// This type represents object sent in a alias call as described in
+// https://segment.com/docs/libraries/http/#alias
+type Alias struct {
+	MessageId  string
```

Comment:
Indeed, thanks for pointing it out. It was missing in the current version of the library but the docs do mention those fields.